src/Core/Addon/Module/ModuleRepository.php

```diff
@@ -422,9 +424,9 @@ private function getAddonsCatalogModules()
      *
      * @return Module
      */
-    public function getModule($name, $skip_main_class_attributes = false)
+    public function getModule($name, $skip_main_class_attributes = false, $force = false)
```

Comment:
```suggestion
    public function getModule($name, $skip_main_class_attributes = false, bool $force = false)
```

Could you add the missing PHPDoc, too ?